### PR TITLE
Add flipper gem and initial setup

### PR DESCRIPTION
**Pull request by eduardoj**
We want to switch the management of the features from the actual [feature](https://github.com/mgsnova/feature) gem to the [flipper](https://github.com/jnunemaker/flipper) gem.

Co-authored-by: David Kang <dkang@suse.com>
Co-authored-by: Henne Vogelsang <hvogel@opensuse.org>
Co-authored-by: Saray Cabrera Padrón <scabrerapadron@suse.de>                                                                                       
Co-authored-by: Victor Pereira <vpereira@suse.com>

### CODE REVIEW

**Comment by saraycp** [comment]
[Add flipper gem & initial setup](https://trello.com/c/QUlAB2wh/1183-add-flipper-gem-initial-setup)

File: src/api/config/initializers/flipper.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Flipper.configure do |config|
+  config.default do
+    adapter = Flipper::Adapters::ActiveRecord.new
+    Flipper.new(adapter)
```
Comment (hennevogel) [inline src/api/config/initializers/flipper.rb]: Hm is this really needed?

Comment (eduardoj) [reply src/api/config/initializers/flipper.rb]: Following [the documentation](https://github.com/jnunemaker/flipper/tree/master/docs/active_record#usage) on how to use the gem with the ActiveRecord adapter, yes.

Comment (hennevogel) [reply src/api/config/initializers/flipper.rb]: Tried it, it replaced the class with an instance in the main namespace...

**Pull request merged**